CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
 # Changelog
 
+## 7.5.0 - 2019/12/02
+
+* 7.5.0 as the default stack version
+* 6.8.5 as 6.x tested version in [#386](https://github.com/elastic/helm-charts/pull/386) [@jmlrt](https://github.com/jmlrt)
+* Helm 2.16.1 support in [#366](https://github.com/elastic/helm-charts/pull/366) [@jmlrt](https://github.com/jmlrt)
+* Add Beats icons to Helm repository in [#345](https://github.com/elastic/helm-charts/pull/345) [@jmlrt](https://github.com/jmlrt)
+* Make helm-tester docker image build less verbose in [#346](https://github.com/elastic/helm-charts/pull/346) [@jmlrt](https://github.com/jmlrt)
+* Update install doc in [#364](https://github.com/elastic/helm-charts/pull/364)[@jmlrt](https://github.com/jmlrt)
```

Comment:
The markdown still renders the links separately but there is a missing space here compared to the other entries.
```suggestion
* Update install doc in [#364](https://github.com/elastic/helm-charts/pull/364) [@jmlrt](https://github.com/jmlrt)
```